.github/workflows/test_plugins.yml

```diff
@@ -0,0 +1,39 @@
+name: Validate neo-modules Compilation with Local neo
+
+on:
+  push:
+    branches: [master]
```

Comment:
Can we add `workflow_dispatch:` trigger as far to be able to run these workflows manually from any branch? It would be really useful for the development process.

Reply:
And we can also add the github commit reference as an input variable to the workflow dispatch so that we could run a specific version of modules/devpack/node (not only master version). We use this approach in NeoGo to be able to build a specific version on any PR. E.g.:
```
  workflow_dispatch:
    inputs:
      ref:
        description: 'Ref to build neo-modules [default: latest master; examples: 3.6.0, 0a4ff9d3e4a9ab432fd5812eb18c98e03b5a7432]'
        required: false
        default: ''
```
And then after that the provided `ref` vaiable can be used with `actions/checkout@v4` to checkout specific commit in neo-modules or devpack or CLI. See also how it works in NeoGo: https://github.com/nspcc-dev/neo-go/blob/f31e3f70aaba07335111a15ee251431e608d8faf/.github/workflows/build.yml#L21

Reply:
i did not learn what v4 can do, but tests ok, so, v4.